website/docs/r/cdn_frontdoor_origin.html.markdown

```diff
@@ -0,0 +1,179 @@
+
+* `origin_host_header` - (Optional) The host header value sent to the origin with each request. Possible values include an `IPv4` IP address, `IPv6` IP address or a valid `domain name`. If you leave field undefined, the request's hostname determines this value. Azure Frontdoor Origins, such as Web Apps, Blob Storage, and Cloud Services require this host header value to match the origin's hostname. This field's value overrides the host header defined in the Frontdoor Endpoint. For more information on how to properly set the origin host header value please see the [product documentation](https://docs.microsoft.com/azure/frontdoor/origin?pivots=front-door-standard-premium#origin-host-header).
+
+* `priority` - (Optional) Priority of origin in given origin group for load balancing. Higher priorities will not be used for load balancing if any lower priority origin is healthy. Must be between `1` and `5`(inclusive). Defaults to `1`.
+
+* `weight` - (Optional) The weight of the origin in a given origin group for load balancing. Must be between `1` and `1000`. Defaults to `500`.
+
+---
+
+A `private_link` block supports the following:
+
+!>**IMPORTANT:** The approval of the Private Link Endpoint is currently a manual step in this process. For more information please see the [product documentation](https://docs.microsoft.com/azure/frontdoor/private-link).
+
+!>**IMPORTANT:** Due to a bug in the service teams code, if you are attempting to destroy a `Private Link Endpoint` connection to a `azurerm_lb`(e.g. `Azure Load Balancer`) resource within your CDN Frontdoor Origin resource which requires you to standup your own `azurerm_private_link_service`. You **MUST** add a `depends_on` attribute to your `azurerm_cdn_frontdoor_profile` which references the `azurerm_private_link_service`(e.g. `depends_on = [azurerm_private_link_service.example]`). If this step is skipped you will receive a `Private link service /subscriptions/{subscription}/resourceGroups/{resourceGroup}/providers/Microsoft.Network/privateLinkServices/{privateLinkService} cannot be deleted since it has N private endpoint connections. Please delete all private endpoint connections before deleting the private link service.` error.
+
+* `request_message` - (Optional) The request message that will be submitted to the `private_link_target_id` when requesting the private link endpoint connection. Values must be between `1` and `140` characters in length. Defaults to `Access request for CDN Frontdoor Private Link Origin`.
+
+* `target_type` - (Optional) `TBD` (`**NOTE:**` as of right now I know these are valid `blob`, `blob_secondary`, `web` or `sites`. This value is not valid for `Load Balancer` resources which you must stand up your own Private Link Service).
+
+* `location` - (Required) The location of the private link resource, for performance reasons this value should match the location of the `private_link_target_id` resource.
```

Comment:
should we add a link to the docs which locations are possible https://docs.microsoft.com/en-us/azure/frontdoor/private-link#region-availability